services/galley/src/Galley/API/Update.hs

```diff
@@ -1114,6 +1115,32 @@ postProteusMessage sender zcon conv msg = runLocalInput sender $ do
     (\c -> postRemoteOtrMessage (qUntagged sender) c (rpRaw msg))
     conv
 
+postProteusBroadcast ::
+  Members
+    '[ BotAccess,
+       BrigAccess,
+       ClientStore,
+       ConversationStore,
+       Error ActionError,
+       Error TeamError,
+       FederatorAccess,
+       GundeckAccess,
+       ExternalAccess,
+       Input Opts,
+       Input UTCTime,
+       MemberStore,
+       TeamStore,
+       TinyLog
+     ]
+    r =>
+  Local UserId ->
+  ConnId ->
+  QualifiedNewOtrMessage ->
+  Sem r (PostOtrResponse MessageSendingStatus)
+postProteusBroadcast sender zcon msg =
+  runLocalInput sender $
```

Comment:
I don't understand why this works. `postBroadcast` does not have a member `Input (Local ())` in its constraints, so where does it come from?